server/config.ts

```diff
@@ -372,6 +372,7 @@ export default {
     editReligionEnabledFrom: get('EDIT_RELIGION_ENABLED_FROM', '2099-01-01T00:00:00'),
     externalContactsEnabledPrisons: get('EXTERNAL_CONTACTS_ENABLED_PRISONS', []),
     manageAllocationsEnabled: toBoolean(get('MANAGE_ALLOCATIONS_ENABLED', 'false')),
+    personEndpointsEnabled: toBoolean(get('PERSON_ENDPOINTS_ENABLED', 'true')),
```

Comment:
obviously i know next to nothing about this, but do you want this on by default in all environments?